public/js/RoomClient.js

```diff
@@ -7295,13 +7295,31 @@ class RoomClient {
         if (inputPv && audioConsumerPlayer) {
             inputPv.style.display = 'inline';
             inputPv.value = 100;
-            // Not work on Mobile?
-            inputPv.addEventListener('input', () => {
-                audioConsumerPlayer.volume = inputPv.value / 100;
-            });
+            
+            const updateVolume = () => {
```

Comment:
My only comment here is that the updateVolume method should be outside of the handlePV as a separate method so that we don't have merge conflicts if the code doesn't exist in mirotalk, just making it easier on the upstream merges we do in the future.

Reply:
separated